infoschema/metric_schema.go

```diff
@@ -925,4 +925,1055 @@ var MetricTableMap = map[string]MetricTableDef{
+		Labels:  []string{"db", "instance", "type"},
+		Comment: "The total number of channel full errors on each TiKV instance, it will make the TiKV instance unavailable temporarily",
+	},
+	"tikv_coprocessor_is_busy": {
+		PromQL:  `sum(rate(tikv_coprocessor_request_error{type='full'}[$RANGE_DURATION])) by (instance,db,type)`,
+		Labels:  []string{"db", "instance"},
+		Comment: "Indicates occurrences of events that make the TiKV instance unavailable temporarily, such as Coprocessor Full",
+	},
+	"tikv_engine_write_stall": {
+		PromQL:  `avg(tikv_engine_write_stall{type="write_stall_percentile99"}) by (instance, db)`,
+		Labels:  []string{"db", "instance"},
+		Comment: "Indicates occurrences of events that make the TiKV instance unavailable temporarily, such as Write Stall",
+	},
+	"tikv_server_report_failures": {
+		PromQL:  `sum(rate(tikv_server_report_failure_msg_total{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance,store_id)`,
+		Labels:  []string{"instance", "store_id", "type"},
+		Comment: "The total number of reporting failure messages",
+	},
+	"tikv_storage_async_requests": {
+		PromQL:  `sum(rate(tikv_storage_engine_async_request_total{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance, status, type)`,
+		Labels:  []string{"instance", "status", "type"},
+		Comment: "The number of different raftstore errors on each TiKV instance",
+	},
+	"tikv_scheduler_stage_total": {
+		PromQL:  `sum(rate(tikv_scheduler_stage_total{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance, stage,type)`,
+		Labels:  []string{"instance", "stage", "type"},
+		Comment: "The number of scheduler state on each TiKV instance",
+	},
+	"tikv_coprocessor_request_error": {
+		PromQL:  `sum(rate(tikv_coprocessor_request_error{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance, reason)`,
+		Labels:  []string{"instance", "reason"},
+		Comment: "The number of different coprocessor errors on each TiKV instance",
+	},
+	"tikv_region_change": {
+		PromQL:  `sum(delta(tikv_raftstore_region_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance,type)`,
+		Labels:  []string{"instance", "type"},
+		Comment: "The count of region change per TiKV instance",
+	},
+	"tikv_leader_missing": {
+		PromQL:  `sum(tikv_raftstore_leader_missing{$LABEL_CONDITIONS}) by (instance)`,
+		Labels:  []string{"instance"},
+		Comment: "The count of missing leaders per TiKV instance",
+	},
+	"tikv_active_written_leaders": {
+		PromQL:  `sum(rate(tikv_region_written_keys_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance)`,
+		Labels:  []string{"instance"},
+		Comment: "The number of leaders being written on each TiKV instance",
+	},
+	"tikv_approximate_region_size": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_raftstore_region_size_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,instance))`,
+		Labels:   []string{"instance"},
+		Quantile: 0.99,
+		Comment:  "The approximate Region size",
```

Comment:
```suggestion
		Comment:  "The approximate P99 Region size",
```

Reply:
Since the `quantile` can be changed by the query, how about adding a default value is P99.